src/data/multiset/powerset.lean

```diff
@@ -210,7 +210,7 @@ congr_arg coe powerset_len_aux_eq_map_coe
   powerset_len 0 s = {0} :=
 quotient.induction_on s $ λ l, by simp [powerset_len_coe']; refl
 
-@[simp] theorem powerset_len_zero_right (n : ℕ) :
+theorem powerset_len_zero_right (n : ℕ) :
```

Comment:
Why this is no longer a `simp` lemma?

Reply:
It was superceded by one of the added ones i think, so the linter complained

Reply:
Yes, this is now proved by `powerset_len_empty`, since `0.card < n + 1` can be proved by `simp`.